app/containers/Sessions/containers/SettingsSessionPage/components/SessionSettings.js

```diff
@@ -183,6 +183,29 @@ const SessionSettings = ({
     });
   };
 
+  const shouldTriggerModal = (selectedNarrator) => {
+    if (currentNarrator === 'peedy' && selectedNarrator !== null) return true;
+    if (
+      (currentNarrator === 'emmi' || currentNarrator === 'crystal') &&
+      selectedNarrator === 'peedy'
+    ) {
+      return true;
+    }
+    if (selectedNarrator === null || currentNarrator === undefined)
+      return false;
```

Comment:
I wonder if this condition is necessary, because after this we also return false